fs/walk.ts

```diff
@@ -31,14 +39,7 @@ function include(
 
 function wrapErrorWithRootPath(err: unknown, root: string) {
   if (err instanceof Error && "root" in err) return err;
-  const e = new Error() as Error & { root: string };
-  e.root = root;
```

Comment:
Can we remove `root` property from the thrown error?

Reply:
We can? It's just not optimal in my opinion. Are we renaming it to `path` instead or completely removing it?

Reply:
`path` sounds good to me